src/live_restore/live_restore_fs.c

```diff
@@ -71,51 +71,6 @@ __live_restore_fs_backing_filename(WTI_LIVE_RESTORE_FS_LAYER *layer, WT_SESSION_
     return (ret);
 }
 
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wunused-function"
-/*
- * __live_restore_debug_dump_extent_list --
- *     Dump the contents of a file handle's extent list. Callers must hold the extent list readlock
- *     at a minimum.
- */
-static void
-__live_restore_debug_dump_extent_list(WT_SESSION_IMPL *session, WTI_LIVE_RESTORE_FILE_HANDLE *lr_fh)
```

Comment:
can we still have an equivalent dump function on error or util cases?

Reply:
That would effectively be calling decode, do we need a dump function that calls that with an additional printf or similar?

Reply:
It could be useful to dump that in a couple of error cases such as:
```
            if (read_begins_in_hole)
                /* A partial read should never begin in a hole. */
                WT_ASSERT_ALWAYS(session, false,
                  "Read (offset: %" PRId64 ", len: %" WT_SIZET_FMT
                  ") begins in a hole but does not end in one (offset: %" PRId64 ")",
                  offset, len, WTI_BIT_TO_OFFSET(current_bit));
```

Reply:
36ec29df1

Reply:
Nice, thanks!